ui/src/app/base/components/Pager/Pager.js

```diff
@@ -0,0 +1,132 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import PagerButton from "app/base/components/PagerButton";
+import PagerItem from "app/base/components/PagerItem";
+
+const PagerItemSeparator = () => (
+  <li className="p-pagination__item p-pagination__item--truncation">…</li>
+);
+
+const Pager = ({
```

Comment:
Could we call this `Pagination` to be consistent with Vanilla?

Reply:
Yep, good call.